extensions/functions_comparison.yaml

```diff
@@ -78,9 +84,15 @@ scalar_functions:
     description: Whether a value is not a number.
     impls:
       - args:
+          - name: value_for_null
```

Comment:
Similarly, what's the motivation to allow NULL to take on different logical meaning?

Reply:
Assuming there is some function that assigns a default value to null values, it's also not primitive. I'm too lazy to look up if it does, but if not it probably should; I'll define it as `unwrap_or(T?, T) -> T` for now (borrowing the name from Rust). In that case, this function is just `is_nan(x)`, `unwrap_or(is_nan(x), true)`, or `unwrap_or(is_nan(x), false)` for the three options respectively, with the added benefit that `value_for_null` can now be any expression.